core_fw/remote_radio/overlay-bt_rpmsg.conf

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+CONFIG_MBOX=y
+CONFIG_IPC_SERVICE=y
+CONFIG_IPC_SERVICE_BACKEND_RPMSG=y
+
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_MAX_CONN=4
```

Comment:
The `CONFIG_BT_MAX_CONN=4` should be applied when building the BLE+802.15.4 combination, but not necessarily when building BLE alone. I don't know what's the correct number in that case, so I think the requirements should be clarified.

Reply:
What about setting this to 16 and adding a comment that it can be reduced as needed?
This config file can be easily overwritten by a config local to a sample using it (the same way other child_image are configured).

Reply:
That would make the BLE+802.15.4 unbuildable by default on nRF53. I don't think it would be good UX to have an unbuildable configuration and require the user to do manual modifications.
For BLE+802.15.4 the default number of connections on nRF53 should be exactly 4, no more, no less.

Reply:
I changed the behavior of the core image. When building hci_rpmsg alone the connection count id 16, but when building with IEEE 802.15.4 it's reduced to 4 as described by you.